server/docs/Recommendation System.md

```
@@ -0,0 +1,523 @@
+CREATE INDEX teaching_chunks_embedding_idx 
+  ON teaching_chunks 
+  USING hnsw (embedding vector_cosine_ops);
+```
+
+-- Index for teaching lookup
+```
+CREATE INDEX teaching_chunks_teaching_id_idx 
+  ON teaching_chunks(teaching_id);
+```
+2. Update teachings table
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)